### ISSUE TITLE
Only /status not working 

### ISSUE THREAD

**Issue by Halzkomy** [cameroncros/OctoPrint-DiscordRemote#190]
I`ve been using Discord remote with my 3D printer for 2 days now. Everything worked fine until today. I shut down my printer and raspberry pi and then turned the power off. Later i started the printer and raspberry pi with Octoprint again. Everything seemed normal. My problem is that everything is working just fine except the command /status. Nothing happens when i type it in discord or try to use it via the Discord Remote symbol in Octoprint. I can pause and resume the print, /help is working and so on. The only thing that seems to not work is /status, which worked fine until today.

Log file:
[plugin_discordremote.log](https://github.com/cameroncros/OctoPrint-DiscordRemote/files/6566067/plugin_discordremote.log)

Screenshot of other commands:
![Screenshot status not working](https://user-images.githubusercontent.com/85059846/120101120-c1e6d700-c144-11eb-8143-b39ff0226d77.PNG)

 - OS: Windows 10 Pro
 - Browser: Chrome

**Comment by cameroncros** [comment]
What was wrong? Did /status start working?

**Comment by Halzkomy** [comment]
Yes, it started working about 1 hour into another print i started.

**Comment by cameroncros** [comment]
hmm, that still sounds like a bug

**Comment by stheyounger** [comment]
This just happened to me too. I finished setting up and only /status doesn't work. Same as Halzkomy's situation.

**Comment by cameroncros** [comment]
Can you please provide any logs?

**Comment by stheyounger** [comment]
[plugin_discordremote.log](https://github.com/cameroncros/OctoPrint-DiscordRemote/files/6719563/plugin_discordremote.log)
